docs/release-notes/list-active-users.rst

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**New Features**
+
+-  CLI: Modify ``det user ls`` ls to show only active users. Add a new flag ``--all`` to show all
+   users.
```

Comment:
The first part sounds like an improvement or a breaking change. The second part sounds like a new feature.

Reply:
i'll update the documentation via PR #8479